src/app/common/DataOverview.vue

```diff
@@ -391,9 +366,12 @@ function getCellAttributes({ headerKey }: any): Record<string, string> {
   return { class: className }
 }
 
-function getRowAttributes({ name }: any): Record<string, string> {
-  const entityName = props.selectedEntityName || props.tableData.data[0].name
-  const className = name === entityName ? 'is-selected' : ''
+function getRowAttributes(row: any): Record<string, string> {
+  if (!row || !row.entity) {
+    return {}
+  }
+
+  const className = row.entity.name === selectedEntityName.value ? 'is-selected' : ''
```

Comment:
Little side question: Do you have any theories on how comes GH can't syntax highlight this code?

Reply:
I think this had to do with the presence of HTML comments that used three dashes (e.g. `<!--- test --->`) like so (it actually reproduces here, too):

```vue
<!--- test --->
<p>Test</p>
```

I’ve removed them now and that fixes it.